athena-gcs/src/test/java/com/amazonaws/athena/connectors/gcs/GcsMetadataHandlerTest.java

```diff
@@ -318,6 +316,46 @@ public void testDoGetSplits() throws Exception
         );
         when(table.getPartitionKeys()).thenReturn(columns);
         GetSplitsResponse response = gcsMetadataHandler.doGetSplits(blockAllocator, request);
-        assertNotNull(response);
+        assertEquals(2, response.getSplits().size());
+        assertEquals(2, response.getSplits().stream().map(split -> split.getProperties().get("year")).count());
+    }
```

Comment:
You are still only checking for the __count__ here, you are not validating that the values of "year" (which are "2000" and "2001").